src/helper/indexeddb.ts

```diff
@@ -0,0 +1,36 @@
+import { openDB } from 'idb';
+
+interface ProjectDB {
+    projects: {
+        key: number; // timestamp as key
+        value: { title: string; createdAt: number };
+    };
+}
+
+export async function initDB() {
+    return await openDB<ProjectDB>('offline-projects-db', 1, {
+        upgrade(db) {
+            if (!db.objectStoreNames.contains('projects')) {
+                db.createObjectStore('projects', { keyPath: 'createdAt' });
+            }
+        },
+    });
+}
+
+export async function saveProject(title: string) {
+    const db = await initDB();
+    const project = { title, createdAt: Date.now() };
+    await db.add('projects', project);
+    console.info('Project saved to IndexedDB:', project);
```

Comment:
These console logs don't serve any purpose for the user and thus should be removed.
This would also elevate the need for mocking the console API during unit testing.